.prettierignore-sync

```diff
@@ -2,7 +2,10 @@
 .gitignore
 
 [relative]
-packages/**/{.gitignore,.prettierignore}
+packages/{*,labs/*}/{.gitignore,.prettierignore}
+# Starter templates specify the lint/format ignore in .eslintignore to stay DRY
+packages/{lit-starter-ts,lit-starter-js}/.eslintignore
```

Comment:
We should move this to the `.eslintignore-sync`, right?
The starters has `.eslintignore` and no `. prettierignore `.

Reply:
The starter kits use their `.eslintignore` as the ignore file for Prettier. This just reflects that.